index.bs

```diff
@@ -6286,6 +6290,58 @@ the wrong [=credential ID=], or if an attacker intercepts and manipulates the [=
+as illustrated in <a href="#fig-proximity-1">Figure <span class="figure-num-following"></span></a>.
+If implementing a solution where the above assumption does not hold,
+implementers SHOULD consider how this affects the [=client=]'s ability to enforce [=scope=] restrictions.
+
+<figure id="fig-proximity-1">
+    <img src="images/authenticator-proximity-1.svg"/>
+    <figcaption>Direct communication between [=client=] and [=authenticator=]
+    enables enforcement of [=credential=] [=scope=] restrictions.</figcaption>
+</figure>
+
+In <a href="#fig-proximity-1">Figure <span class="figure-num-previous"></span></a>,
+the [=client=] and [=authenticator=] communicate directly.
+If the user visits a malicious site (1), then the [=client=] can refuse to initiate an [=authentication ceremony=]
+if the site's [=origin=] does not match the claimed [=RP ID=].
+On the other hand, if the malicious application uses an [=RP ID=] valid for its [=origin=],
+then the [=authentication assertion=] (2) will be signed over that [=origin=] instead of the [=[RP]=]'s [=origin=],
+and the [=[RP]=] can reject the mismatched [=origin=] when the malicious actor attempts
+to use the [=authentication assertion=] to [=authentication|authenticate=] to the [=[RP]=] (3).
+
+If the [=client=] does not communicate with the [=authenticator=] directly,
+then enforcing the [=scope=] restrictions might be more challenging.
+
+<figure id="fig-proximity-2">
+    <img src="images/authenticator-proximity-2.svg"/>
+    <figcaption>Indirect communication between [=client=] and [=authenticator=]
+    might allow malicious actors to bypass [=scope=] restrictions.</figcaption>
+</figure>
+
+In <a href="#fig-proximity-2">Figure <span class="figure-num-previous"></span></a>,
+ the [=client=] communicates with the [=authenticator=] indirectly via some external authenticator service (1, 2).
+For example, the [=client=] might send an HTTP request to a remote server (1)
+which then sends a push notification (2) to the user's mobile device,
+which prompts for [=user consent=] and acts as the [=authenticator=].
+If the initial request (1) is not authenticated,
+it might be possible for a malicious actor to trigger (3) the [=authenticator=] prompt (2),
+receive the [=authentication assertion=] and use it to authenticate to the [=[RP]=] (4).
+
+In order to enforce the [=scope=] restrictions in this example,
+the [=client=] would need to authenticate itself to the authenticator service (1)
+before the authenticator service allows access to the user's [=authenticator=] (2).
+This would ensure that the communication with the [=authenticator=]
+is mediated by the user's [=client=] so that the [=client=] can enforce the [=credential=] [=scope=] restrictions.
+
+Implementers of similar protocols SHOULD ensure that the [=client=] can still enforce [=credential=] [=scope=] restrictions
+despite not communicating with the [=authenticator=] directly.
```

Comment:
s/ directly. / directly, or they should establish the means to create subsequent secure channels between the client and authnr via some initial channel that is not vulnerable to MITM attacks. E.g., exchanging pre-shared keys between the client and authnr over a direct physical USB connection or similar. /

Reply:
Nice, thanks! I rearranged the sentence a structure a bit to make it less ambiguous what left-hand-side the "via" refers to.